packages/server/lib/browsers/cdp-command-queue.ts

```diff
@@ -0,0 +1,83 @@
+import type ProtocolMapping from 'devtools-protocol/types/protocol-mapping'
+import pDefer, { DeferredPromise } from 'p-defer'
+import type { CdpCommand } from './cdp_automation'
+import Debug from 'debug'
+
+const debug = Debug('cypress:server:browsers:cdp-command-queue')
```

Comment:
How chatty will these logs be? Do we want to make some of them verbose and some of them non-verbose?

Reply:
Not very chatty - commands are only pushed into the command queue when they come in while the websocket connection is unexpectedly disconnected. params might be a larger payload, though, and could benefit from a verbose (or removal, as it's not pertinent to the command queue operations)

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/29830/commits/fc5203300d839750a987ab417bf5ac6d812dbae6